core/src/main/java/com/linecorp/armeria/server/file/HttpFileServiceBuilder.java

```diff
@@ -98,11 +103,26 @@ public HttpFileServiceBuilder clock(Clock clock) {
     }
 
     /**
-     * Sets the maximum allowed number of cached file entries. If not set, {@value #DEFAULT_MAX_CACHE_ENTRIES}
+     * Sets the maximum allowed number of cached file entries. If not set, {@code "maximumSize=1024"}
      * is used by default.
      */
     public HttpFileServiceBuilder maxCacheEntries(int maxCacheEntries) {
-        this.maxCacheEntries = HttpFileServiceConfig.validateMaxCacheEntries(maxCacheEntries);
+        checkState(canSetMaxCacheEntries,
+                   "Cannot call maxCacheEntries() if called entryCacheSpec() already.");
+        entryCacheSpec = HttpFileServiceConfig.validateMaxCacheEntries(maxCacheEntries);
+        canSetEntryCacheSpec = false;
+        return this;
+    }
+
+    /**
+     * Sets the cache spec for caching file entries. If not set, {@code "maximumSize=1024"} is used by default.
+     */
+    public HttpFileServiceBuilder entryCacheSpec(String entryCacheSpec) {
```

Comment:
I just saw that code patch coverage is lower than the baseline. Could you add a test case for here :-)

Reply:
I added test case. Please tell me if it is not enough.